core/domain/user_services.py

```diff
@@ -196,8 +196,8 @@ def normalize_username(cls, username):
             username: str. Identifiable username to display in the UI.
 
         Returns:
-            str or None. A string representing normalized username if it exists in this
-            UserSettings domain object. Otherwise None.
+            str or None. A string representing normalized username if user with
+            given username exists. Otherwise None.
```

Comment:
This is still misleading; the method implementation does not actually check whether a user with the given username exists. Instead:

str or None. The normalized version of the given username, or None if the passed-in `username` is None.